Ukraine Cities

```diff
@@ -0,0 +1,103 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+class Program
+{
+    static Dictionary<string, Dictionary<string, int>> cities = new Dictionary<string, Dictionary<string, int>>()
+    {
+        { "Kyiv", new Dictionary<string, int>{{"Chernihiv", 140}, {"Zhytomyr", 140}, {"Poltava", 340}}},
+        { "Chernihiv", new Dictionary<string, int>{{"Kyiv", 140}, {"Sumy", 160}}},
+        { "Zhytomyr", new Dictionary<string, int>{{"Kyiv", 140}, {"Vinnytsia", 120}}},
+        { "Poltava", new Dictionary<string, int>{{"Kyiv", 340}, {"Kharkiv", 140}}},
+        { "Vinnytsia", new Dictionary<string, int>{{"Zhytomyr", 120}, {"Khmelnytskyi", 120}}},
+        { "Sumy", new Dictionary<string, int>{{"Chernihiv", 160}, {"Kharkiv", 180}}},
+        { "Kharkiv", new Dictionary<string, int>{{"Poltava", 140}, {"Sumy", 180}}},
+        { "Khmelnytskyi", new Dictionary<string, int>{{"Vinnytsia", 120}, {"Lviv", 260}}},
+        { "Lviv", new Dictionary<string, int>{{"Khmelnytskyi", 260}, {"Ternopil", 130}}},
+        { "Ternopil", new Dictionary<string, int>{{"Lviv", 130}}}
+    };
+
+    static (List<string> path, int totalDistance) FindShortestPath(string start, string end)
```

Comment:
Use custom models for path and distance, for instance a separate class. That would be good to derive class `Path` from `List<string>`.
Also, distance could be non-integer 12.342kM for example. Use `decimal` values